repo/packages/D/dcos-ui/0/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "packagingVersion": "4.0",
+  "name": "dcos-ui",
+  "version": "1.0.0",
```

Comment:
Which version should we take here? Should it be the current one? (Not in this PR, but more of a general question?)